release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -209,6 +208,17 @@ If you have already upgraded your cluster to {product-title} {product-version} a
 
 For more information, see xref:../networking/ovn_kubernetes_network_provider/converting-to-dual-stack.html#nw-dual-stack-convert_converting-to-dual-stack[Converting to a dual-stack cluster network] in _Converting to IPv4/IPv6 dual-stack networking_.
 
+[id="ocp-4-16-networking-changing-ovn-kubernetes-network-plugin-internal-range_{context}"]
+==== Support for changing the OVN-Kubernetes network plugin internal IP address ranges
+
+If you use the OVN-Kubernetes network plugin, you can configure the transit, join, and masquerade subnets. The former two subnets can be configure either during cluster installation or after. The masquerade subnet must be configured during installation and cannot be changed after. The subnet defaults are:
```

Comment:
Just to be clear, maybe add the subnet names instead of _former two_.
```suggestion
If you use the OVN-Kubernetes network plugin, you can configure the transit, join, and masquerade subnets. The transit and join subnets can be configured either during cluster installation or after. The masquerade subnet must be configured during installation and cannot be changed after. The subnet defaults are:
```